### DISCUSSION TITLE
Too slow load page/rendering

### DISCUSSION THREAD

**Discussion by maxreig** [requarks/wiki#5319] [Error / Bug Report]
Hi everyone,
I have had WIKI JS in production for a week and I am observing that the page loading time is increasing...
Can you help me
Kind  Regards,

**Comment by fumbles** [comment]
What's the number of total docs you have? I noticed ours gets slower also... have 58k docs though

**Comment by crustymonkey** [comment]
I spun up an instance of Wiki.js in a container with only 1 edit to the main doc and I've noticed that it takes ~400ms (or more) to just get a server response (according to chrome dev tools and curl).  This is running in a k8s container fronted by Envoy and everything else in the cluster is fast.  This is using their official `requarks/wiki` docker release too.

**Reply by fumbles** [reply]
Unsure about that specifically. Could be a lot of things- storage delays.

I did post a patch a while back.
https://github.com/requarks/wiki/pull/6082
What works for us

[patches.zip](https://github.com/requarks/wiki/files/13314296/patches.zip)

![image](https://github.com/requarks/wiki/assets/90968/4d30db81-59d0-45ba-9600-cde17e75c896)

**Reply by crustymonkey** [reply]
Yeah, storage shouldn't be an issue as that's an NFS PVC about 1ms away.  I have a number of other containers using the same NFS provider with no issues.  One thing I didn't note in the original post is that this is running arm64, not x86_64.

I'm probably going to try just building an image  from scratch and see if that performs any differently.  I'll also check out your patches.

**Reply by enricocirignaco** [reply]
Does it make a difference when running on x86_64? I'm also on arm64 and rendering (even of small pages) is very slow ~2-3 seconds
